src/Compilers/CSharp/Portable/Binder/Binder_Operators.cs

```diff
@@ -2550,6 +2550,20 @@ private bool IsOperandErrors(CSharpSyntaxNode node, ref BoundExpression operand,
                     }
 
                     return true;
+
+                default:
+                    if ((object)operand.Type == null && !operand.IsLiteralNull())
+                    {
+                        if (!operand.HasAnyErrors)
+                        {
+                            // Operator 'is' cannot be applied to operand of type '(int, <null>)'
+                            diagnostics.Add(ErrorCode.ERR_BadUnaryOp, node.Location, "is", operand.Display);
```

Comment:
>"is" [](start = 85, length = 4)

There isn't a single place in the codebase where we hardcode textual representation of a token while reporting ERR_BadUnaryOp error, please use SyntaxFacts.GetText(SyntaxKind) instead.